netsim/modules/vlan.yml

```diff
@@ -55,13 +55,17 @@ attributes:
     bandwidth:
     _selfloop_ifindex:
     stp:
+    virtual_interface:  # Use case: VLAN on lag interface
+    lag:                # Keep lag parameters such as lacp settings
```

Comment:
Are you telling me you never tested LAG with VLAN interfaces before saying "the LAG module is ready"? Otherwise, I agree this change is needed as LAG is below VLANs in the IEEE protocol stack.

Reply:
I did test lags with vlans: ```netlab create integration/lag/02-lag-vlan-trunk.yml -p clab -d frr```

Validation passes on ```dev``` - but I agree that apparently the test wasn't rigorous enough.

When I add this patch, the only thing that changes in that 02 integration test, is that ```vlan_member``` interfaces are no longer marked as ```virtual_interface```. Adding that under ```keep_subif``` fixes that (that is why I added that)

I did try adding ```virtual_interface``` as an interface attribute in ```defaults/attributes.yml``` and setting it in ```create_vlan_member_interfaces``` but that didn't work. I figured it's essentially part of the ```type```, so copying it along with the ```type``` attribute makes some sense